prediction/src/tests/test_identify_trained_model_predict.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from ..preprocess import preprocess_dicom
+from ..algorithms.identify import trained_model
+
+@pytest.fixture
+def dicom_path():
+    yield '../images/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192'
+
+
+
+def test_classify_predict_model_load(dicom_path):
```

Comment:
As @lamby already stated [here](https://github.com/concept-to-clinic/concept-to-clinic/pull/118#issuecomment-330149196) it'd be nice to still have some tests that validate the output of the models. I'm currently working on some tags to only execute these long running tests if one explicitly desires to do so. So feel free to write this test a bit more detailed :)

Reply:
Sure, I'll have a look at that tomorrow.

Reply:
I updated the test cases to be a little more comprehensive.